### PR TITLE
Added traditional Chinese support

### PR DESCRIPTION
**Pull request by vicary**
Numbers up to 100

### CODE REVIEW

File: src/timeAgo.js [hunk 1]
```diff
@@ -21,21 +21,17 @@ angular.module('yaru22.angular-timeago', [
...
-}]).factory('nowTime', ['$window', '$rootScope', function ($window, $rootScope) {
-  var nowTime = Date.now();
-  var updateTime = function() {
-    $window.setTimeout(function() {
-      $rootScope.$apply(function(){
-        nowTime = Date.now();
-        updateTime();
-      });
-    }, 1000);
+}]).factory('nowTime', function ($timeout) {
```
Comment (yaru22) [inline src/timeAgo.js]: Could you make this change a separate PR? Want this PR to be strictly about Chinese language support.

Comment (vicary) [inline src/timeAgo.js]: I've never done a rollback before, should I make a hard reset like this?

```
git reset --hard bfdb8f46b56fa57b916d5c469cfbdcaeafb54dc6
git push -f origin patch-1
```

Comment (yaru22) [inline src/timeAgo.js]: hm.. I haven't done that either. The most straightforward way to do this is making changes back to the original for this part and make another commit. So you will have total 3 commits for this PR.

**Comment by vicary** [comment]
@yaru22 Consider it done. Btw would you like to take the latter commit as a separate PR?

**Comment by yaru22** [comment]
@vicary sorry for the delay. Totally slipped through my mind. If you'd like to add the latter commit as a separate PR, that's cool, too. Thanks for the contribution!
